Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```
@@ -1007,7 +1007,39 @@
   recipeType:
   - Oven
 
-# NOT ACTUAL FOOD
+- type: microwaveMealRecipe
+  id: RecipeBunCotton
+  name: cotton bun recipe
+  result: FoodBunCotton
+  time: 5
+  group: Breads
```

```suggestion
  group: Moth
```
group dictates where it shows up in the guidebook